### ISSUE TITLE
Make geordi work with multiline server declaration

### ISSUE THREAD

**Issue by PragTob** [makandra/geordi#44]
works:

```ruby
server "app101-prod.totally-not-makandra.de", user: "deploy_p", roles:   %w[app web db cron], primary: true
```

breaks:

```ruby
server "app101-prod.totally-not-makandra.de",
       user:    "deploy_p",
       roles:   %w[app web db],
       primary: true
```

i.e. `geordi shell/console` break because `@config.user(server)` is nil. Like in `Util#shell`:

```ruby
      note 'Connecting to ' + server.to_s
      Util.system! 'ssh', "#{ @config.user(server) }@#{ server }", '-t', remote_command
```

If it's easy to fix (which I hope) I might tackle it myself, for now I'll revert my neatly aligned and 80 char width configuration :)

**Comment by triskweline** [comment]
We're currently [grepping the deployment config files](https://github.com/makandra/geordi/blob/master/lib/geordi/capistrano_config.rb) to avoid booting the bundle and Capistrano into our process.

You might be able to fix this by adding a `/m` modifier to the regexps in the quoted file, to make `.` match newlines.

**Comment by denzelem** [comment]
Fixed in #51 and included in release `1.10.0`.
